sherlock_project/resources/data.json

```
@@ -1689,6 +1689,12 @@
     "urlMain": "https://www.rajce.idnes.cz/",
     "username_claimed": "blue"
   },
+  "Rarible.com": {
```

Seems that the site simply goes by Rarible rather than Rarible.com. Would that be correct?
```suggestion
  "Rarible": {
```
Otherwise, tests pass and it'll be good to merge